brain/task_executor.py

```
@@ -405,6 +629,23 @@ async def analyze_and_execute(
                 reason=cu_decision.reason
             )
         
+        # 3. 如果 MCP 不行，但 UserPlugin 可用且可执行，优先调用 UserPlugin
+        if up_decision and getattr(up_decision, "has_task", False) and getattr(up_decision, "can_execute", False):
+            logger.info(f"[TaskExecutor] ✅ Using UserPlugin: {up_decision.task_description}, plugin_id={getattr(up_decision, 'plugin_id', None)}")
+            try:
+                return await self._execute_user_plugin(task_id=task_id, up_decision=up_decision)
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)